src/utils.js

```diff
@@ -38,7 +38,7 @@ const utils = {
   async fetch(uri, logger) {
     if (uri.charAt(0) === '/') {
       try {
-        return await fs.readFile(uri);
+        return await fs.readFile(uri.replace(/\?.*$/, ''));
```

Comment:
i would probably be better to just pass in the path and not the entire uri

Reply:
`uri` is the path in this case.

Reply:
Do you mean to change `uri` to `uriOrPath` in line 38?

Reply:
@trieloff Changed uri to uriOrPath